lib/widgets/message_list.dart

```diff
@@ -1954,6 +1954,10 @@ class _SenderRow extends StatelessWidget {
                           : designVariables.title,
                       ).merge(weightVariableTextStyle(context, wght: 600)),
                       overflow: TextOverflow.ellipsis)),
+                  if (sender != null)
+                    Padding(
+                      padding: const EdgeInsetsDirectional.only(start: 5.0),
+                      child: UserStatusEmoji(userId: sender.userId, size: 18)),
```

Comment:
We don't actually need the User object here, right? So we can express the user ID the same way as in the neighboring code:
```suggestion
                  Padding(
                    padding: const EdgeInsetsDirectional.only(start: 5.0),
                    child: UserStatusEmoji(userId: message.senderId, size: 18)),
```

Reply:
One awkward thing that points at, though, is that we're including the padding even when there's no status emoji to show.

The consequence of that is pretty subtle here. One, it means if there isn't room horizontally for the name, it'll wrap or truncate a bit sooner. Two, if the sender is a bot, there's this extra spacing before the bot icon.

Seems like this might come up in other contexts too, though. Perhaps push this padding inside UserStatusEmoji? The caller can perhaps pass the `padding` argument, exactly like it would to `Padding`; but then if there's no emoji to show, it doesn't get padded.